_data/changelogs/component-memorable-date.yml

```diff
@@ -2,6 +2,16 @@ title: Memorable date
 type: component
 changelogURL:
 items:
+- date: NNNN-NN-NN
+  summary: Removed numeric representation of months from the `select` element.
+  summaryAdditional: Recent usability testing indicated that having both numbers and names to represent months was confusing for screen reader users.
+  isBreaking: true
```

Comment:
**issue (blocking):** I _believe_ we wouldn't mark this as a breaking change in the description, since nothing will technically break if the user doesn't change anything. 

```suggestion

```